packages/docusaurus-plugin-structurizr/src/run-structurizr.ts

```diff
@@ -9,18 +9,18 @@ import type { InternalDocusaurusPluginStructurizrOptions } from './validate-opti
  */
 export async function runStructurizr(
   file: string,
+  docsPath: string,
   options: Pick<
     InternalDocusaurusPluginStructurizrOptions,
     'executor' | 'dockerImage' | 'format' | 'additionalStructurizrArgs'
   >,
 ) {
```

Comment:
**suggestion**: When a function has more than two parameters I prefer prefer object parameters instead of indexed parameters. In this case run `runStructurizr` function does something with a `file` and uses all options to figure out what to do with it. 

This leads me to suggesting that the `docsPath` should be part of the `options`:

```suggestion
export async function runStructurizr(
  file: string,
  options: Pick<
    InternalDocusaurusPluginStructurizrOptions,
    'executor' | 'dockerImage' | 'format' | 'additionalStructurizrArgs'
  > & { docsPath: string },
) {
```

> \*the suggestion is not prettified